daml-foundations/daml-ghc/language-server/src/DA/Service/Daml/LanguageServer.hs

```diff
@@ -244,69 +243,52 @@ handleNotification lspFuncs (IHandle stateRef loggerH compilerH _notifChan) = \c
 -- Server execution
 ------------------------------------------------------------------------
 
-runLanguageServer  :: (  Maybe (Compiler.Event -> IO ())
-                      -> Logger.Handle IO
-                      -> Managed.Managed (VFSHandle -> IO Compiler.IdeState)
-                      )
-                   -> Logger.Handle IO
-                   -> IO ()
-runLanguageServer handleBuild loggerH = Managed.runManaged $ do
+runLanguageServer
```

Comment:
This type signature looks all kinds of odd. I guess it will die in a future version when we get rid of the language server? (To be clear, it looked really odd before, this patch makes it more sane)

Reply:
Yep, that's fine.